snippets/UUIDGenerator.md

```diff
@@ -4,13 +4,26 @@ Generates a UUID.
 
 Use `crypto` API to generate a UUID, compliant with [RFC4122](https://www.ietf.org/rfc/rfc4122.txt) version 4.
 
+Please note that this version will only work in **Node.JS** environments!
```

Comment:
incorrect. Browsers also exposes a global crypto

Reply:
Not what I meant. There are 2 different functions to do the same thing in different platforms. Web has getRandomValues, while Node.JS has randomBytes

Reply:
Maybe we should actually add a Node version of this snippet then, under the **Node** category, this one was for browser, so maybe we can change the tag for this, too and keep them separate. That sounds like a better plan!

Reply:
Fair. Had not remembered that. We may want to note the differences instead of having two different examples. I'd suggest a browser version with a note on the changes for a nodeJS version since most nodeJS developers probably just use the node-uuid package since that is tried and true